src/Ilios/CliBundle/Command/InstallUserZeroCommand.php

```diff
@@ -0,0 +1,133 @@
+     * @var string
+     */
+    const LAST_NAME = 'Zero';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('form:install_user_zero')
+            ->setDescription('Creates a first user account with "Course Director" privileges.')
+            ->addOption(
+                'school',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'A valid school id.'
+            )
+            ->addOption(
+                'email',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'A valid email address.'
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        /**
+         * @var UserManagerInterface $userManager
+         */
+        $userManager = $this->getContainer()->get('ilioscore.user.manager');
+
+        // prevent this command to run on a non-empty user store.
+        $existingUser = $userManager->findUserBy([]);
+        if (! empty($existingUser)) {
+            throw new \Exception('Sorry, at least one user record already exists. Cannot create a "user zero".');
+        }
+
+        /**
+         * @var SchoolManagerInterface $schoolManager
+         */
+        $schoolManager = $this->getContainer()->get('ilioscore.school.manager');
+
+        /** @var FormHelper $formHelper */
+        $formHelper = $this->getHelper('form');
+        $formData = $formHelper->interactUsingForm(
+            new InstallUserZeroType($schoolManager),
+            $input,
+            $output
+        );
+
+        $user = $userManager->createUser();
+        $user->setFirstName(self::FIRST_NAME);
+        $user->setMiddleName(date('Y-m-d_h.i.s'));
+        $user->setLastName(self::LAST_NAME);
+        $user->setEmail($formData['email']);
+        $user->setAddedViaIlios(true);
+        $user->setEnabled(true);
+        $user->setUserSyncIgnore(false);
+        /**
+         * @var UserRoleManagerInterface $userRoleManager
+         */
+        $userRoleManager = $this->getContainer()->get('ilioscore.userrole.manager');
+        $user->addRole($userRoleManager->findUserRoleBy(['title' => 'Course Director']));
+        $user->setSchool($schoolManager->findSchoolBy(['id' => $formData['school']]));
+        $userManager->updateUser($user);
+
+        /**
+         * @var AuthenticationManagerInterface $authenticationManager
+         */
+        $authenticationManager = $this->getContainer()->get('ilioscore.authentication.manager');
+        $authentication = $authenticationManager->createAuthentication();
+
+        $authentication->setUser($user);
+        $user->setAuthentication($authentication); // circular reference needed here. 123 BLEAH! [ST 2015/08/31]
+
+        $encoder = $this->getContainer()->get('security.password_encoder');
+        $encodedPassword = $encoder->encodePassword($user, self::PASSWORD);
```

Comment:
no particular reason at all. again, i just ported this from ilios2 in a rather uninspired fashion.

going forward, i'd like to replace this half-assed solution with a more full-featured suite of user management command line tools.